dependency-check-suppressions.xml

```diff
@@ -105,7 +105,7 @@
         <cve>CVE-2020-8908</cve>
     </suppress>
 
-    <suppress until="2021-01-20">
+    <suppress>
```

Comment:
Do we need to remove this?

Reply:
Post this until date again facing CVE suppression issue.